amplify/backend/function/fetchDomainBalance/src/api/graphql/schemas.js

```diff
@@ -119,15 +155,10 @@ module.exports = {
               tokenAddress
             }
           }
-          fundingActions: actions(filter: { type: { eq: MOVE_FUNDS } }) {
+          fundingActions: actions(
```

Comment:
Maybe we should rename this into `createExpenditureActions` 
And maybe also leave a comment on this line that we'll take into account also the `MOVE_FUNDS` actions needed for expenditures in case there are not enough funds in the domain where the expenditure was created 
`let actions = await getAllActions(colonyAddress, domainId);`